Example/BitmovinAnalyticsCollector/Info.plist

```diff
@@ -31,6 +31,8 @@
 	<string>LaunchScreen</string>
 	<key>UIMainStoryboardFile</key>
 	<string>Main</string>
+	<key>BitmovinPlayerLicenseKey</key>
+	<string>a6e31908-550a-4f75-b4bc-a9d89880a733</string>
```

Comment:
Not sure if you want the player license key here?

Reply:
The sample will crash without an error message when opening the BitmovinPlayer page if the plist doesn't contain a player key - took me ages to figure that out, Player team is aware of it and is working on a proper error message.